packages/salesforcedx-vscode-core/src/commands/auth/orgLoginWeb.ts

```diff
@@ -141,9 +150,44 @@ class OrgLoginWebExecutor extends SfCommandletExecutor<AuthParams> {
 
     return command.build();
   }
+
```

Comment:
I don't understand why this is added.  Isn't it the same as what's inherited from the SfCommandletExecutor?